hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -90,19 +91,63 @@
    * Total number of log records that were compacted by a compaction operation
    */
   @Nullable
-  private Long totalLogRecords;
+  private long totalLogRecords;
 
   /**
-   * Total number of log files that were compacted by a compaction operation
+   * Total number of log files for a file slice with this base fileid
    */
   @Nullable
-  private Long totalLogFiles;
+  private long totalLogFiles;
+
+  /**
+   * Total size of all log files for a file slice with this base fileid
+   */
+  @Nullable
+  private long totalLogFilesSize;
 
   /**
    * Total number of records updated by a compaction operation
    */
   @Nullable
-  private Long totalRecordsToBeUpdate;
+  private long totalRecordsToBeUpdate;
+
+  /**
+   * Total number of log blocks seen in a compaction operation
+   */
+  @Nullable
+  private long totalLogBlocks;
+
+  /**
+   * Total number of corrupt blocks seen in a compaction operation
+   */
+  @Nullable
+  private long totalCorruptLogBlock;
+
+  /**
+   * Total number of rollback blocks seen in a compaction operation
+   */
+  private long totalRollbackBlocks;
+
+  /**
+   * Total time taken to read and merge logblocks in a log file
+   */
+  @Nullable
+  @JsonIgnore
+  private long totalTimeTakenByLogScanner;
+
+  /**
+   * Total time taken by a Hoodie Merge for an existing file
+   */
+  @Nullable
+  @JsonIgnore
+  private long totalTimeTakenForUpsert;
```

Comment:
@vinothchandar they are jsonignore so they won't be serialized to disk. Still see issues ?

Reply:
Lets pull these into a separate class , say `PerfStat` or sth,  as a member in HoodieWriteStat?  It will be easier to remove later on..